mozilla-detekt-rules/src/main/java/org/mozilla/fenix/detektrules/MozillaRunBlockingCheck.kt

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.detektrules
+
+import io.gitlab.arturbosch.detekt.api.*
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.resolve.BindingContext
+import org.jetbrains.kotlin.resolve.calls.callUtil.getCall
+import org.jetbrains.kotlin.resolve.calls.callUtil.getCalleeExpressionIfAny
+import kotlin.math.exp
+
+private const val VIOLATION_MSG = "Please use `org.mozilla.fenix.perf.RunBlockingCounter.runBlockingIncrement`" +
```

Comment:
It's `org.mozilla.fenix.perf.runBlockingImplement` – top level functions don't include the file name in their fully-qualified class name.